.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+version: 2.1
+jobs:
+  build:
+    working_directory: ~/scruid
+
+    docker:
+      - image: mmartina/docker-druid
```

Comment:
Did you change anything in this docker image, compared to @Fokko 's one?
Should we consider creating a ing-bank specific docker-druid fork, to make it futureproof?

Reply:
It appears its just the setting of limiting the nr of threads, since you reverted all the other stuff we worked on yesterday :)

Reply:
So to give a bit of context to @Fokko , in CircleCI2 the Druid container probably dynamically defined how much threads it could use. The calculated number resulted in a lot of memory usage. We restricted the nr of threads to `1`

Reply:
here is the changelog for the docker image:
https://github.com/mmartina/docker-druid/blob/master/CHANGES.md

Reply:
Makes perfect sense. This will reduce the preallocated memory buffers, and therefore reduce the memory footprint. Would be nice to move this to an ING repository in the end.